test/fixtures/render/content-type.apib

```diff
@@ -0,0 +1,14 @@
+# GET /
++ Response 200 (application/json; charset=utf-8)
+    + Attributes
+        + user: pksunkara
+
+# GET /hal
++ Response 200 (application/hal+json)
+    + Attributes
+        + user: pksunkara
+
+# GET /typo
++ Response 200 (application/haljson)
```

Comment:
Do you mean we should support anything ending with `+json`? Currently I think the regex supports `application/*+json`